lib/cirro_io/client/version.rb

```diff
@@ -1,7 +1,7 @@
 # rubocop:disable Style/MutableConstant
 module CirroIO
   module Client
-    VERSION = '1.6.1'
+    VERSION = '1.7.0'
```

Comment:
```suggestion
    VERSION = '1.6.2'
```

Neither the API nor inner logic are changed.

Reply:
just wanted to make sure the version is very different to not confuse with 1.6.1 as it could break the current clients.

Reply:
I don't think this can break clients (unless we made mistakes of course) and therefore should indeed be `1.6.2`